aws/src/integration/java/org/apache/iceberg/aws/glue/GlueTestBase.java

```diff
@@ -114,7 +114,7 @@ public static void beforeClass() {
   @AfterAll
   public static void afterClass() {
     AwsIntegTestUtil.cleanGlueCatalog(GLUE, NAMESPACES);
-    AwsIntegTestUtil.cleanS3Bucket(S3, TEST_BUCKET_NAME, TEST_PATH_PREFIX);
+    AwsIntegTestUtil.cleanS3GeneralBucket(S3, TEST_BUCKET_NAME, TEST_PATH_PREFIX);
```

Comment:
Why is this changing? Surely we want this to be a toggle depending on what the bucket is?

Reply:
I can add the toggle here, I haven't run the tests with glue setup so didn't want to alter the behavior here really. When you were doing your investigation did you also run the glue tests?

Reply:
I didn't - only just noticed this was for general buckets rather than S3 express ones so it's probably fine to leave as-is